solver/base/kustomization.yaml

```diff
@@ -6,7 +6,3 @@ commonLabels:
   app.kubernetes.io/managed-by: aicoe-thoth-devops
 resources:
   - imagestreams.yaml
```

Comment:
All changes in base needs to remain as before, otherwise we break what is in stage already

Reply:
ping @bissenbay

Reply:
thanks, https://github.com/thoth-station/thoth-application/pull/482/commits/eb866721ff12944dd10db2c7b72d0cdb452116e0